assemblies/assembly_openshift-immutable/main.adoc

```diff
@@ -11,31 +11,39 @@ include::_artifacts/author-group.adoc[]
 
 // Purpose statement for the assembly
 As a system engineer, you can deploy a {PRODUCT} immutable server environment on {OPENSHIFT}
-to provide an infrastructure to execute processes and other business assets. You can use
+to provide an infrastructure to execute {ASSETS} and other business assets. You can use
 standard integration tools to manage the immutable {KIE_SERVER} image. You can create new server images
-to add and update processes.
+to add and update {ASSETS}.
 
 .Prerequisites
 
-* At least four gigabytes of memory are available in the OpenShift environment.
+* At least four gigabytes of memory are available in the OpenShift cluster/namespace.
 * The OpenShift project for the deployment is created.
-* You have logged in to the project using the OpenShift web console and using the `oc` command.
-* If you intend to scale any of the {CENTRAL} or {CENTRAL} Monitoring pods, your OpenShift environment supports persistent volumes with ReadWriteMany mode.
-+
-IMPORTANT: ReadWriteMany mode is not supported on OpenShift Online and OpenShift Dedicated.
-
+* You have logged in to the project using the OpenShift web console and using the `oc` command. For more information about the `oc` command-line tool, see the OpenShift https://access.redhat.com/documentation/en-us/openshift_container_platform/3.10/html-single/cli_reference/[CLI Reference].
+ifeval::["{PRODUCT_INIT}"=="rhpam"]
+* Dynamic persistent volume (PV) provisioning is enabled. Alternatively, enough persistent volumes are available. By default, the following sizes are required:
+** Each immutable server deployment includes a replicated set of {KIE_SERVER} pods, which requires one 1Gi PV for the database. You can deploy multiple immutable servers; each requires a separate 1Gi PV. 
```

Comment:
it is not a requirement, is a customizable field.
this requirement is only for OpenShift Online and was decided to use this value.

Reply:
Changed throughout.